.gitmodules

```diff
@@ -19,3 +19,6 @@
 	path = melonds
 	url = https://github.com/TASVideos/melonDS.git
 	branch = SuuperW_DSHawk
+[submodule "waterbox/musl"]
```

Comment:
@adelikat @vadosnaprimer Do we want to sobmodule LLVM?  We have not made any modifications to it, nor do we plan to, and it would increase the size of a full recursive checkout massively.

Reply:
I care about having a fork we'd independently work with, in this organization. If it's problematic to have the entire thing submoduled, releasing our fork's binaries and using them here may be a way to go?

Reply:
I would rather see a separate repository, which has bizhawk as a submodule and also contains all the ginormous build parts. This way the build parts can count on emitting dlls into the appropriate location in the bizhawk directory for testing/

Reply:
Just got an even better idea:  Let's see what happens when other people try to recreate the environment.